### DISCUSSION TITLE
Virtualized x86_64 on Apple Silicon (what am I missing?)

### DISCUSSION THREAD

**Discussion by myshkin5** [utmapp/UTM#5707] [UTM Help (macOS)] [answered]
1. Create a New Virtual Machine
2. Virtualize
3. Linux
4. Checked "Enable Rosetta (x86_64 Emulation)"
5. Browsed to a downloaded copy of `ubuntu-22.04.3-live-server-amd64.iso`
6. Ignored this message (x86_64 virtualization via Rosetta does work, right?):
    ![image](https://github.com/utmapp/UTM/assets/6935773/ba497b46-8a7f-456a-9ffb-31a9725c0822)
7. Hit play -- nothing but a black screen

I must be missing something, right?

**Comment by osy** [comment]
> Ignored this message

🫤

https://docs.getutm.app/guides/debian/

**Reply by myshkin5** [reply]
@osy So what does the "Enable Rosetta (x86_64 Emulation)" checkbox actually do? It seems I can run an arm64 distro with or without selecting this checkbox, right?

**Reply by jepefe** [reply, marked as answer]
It enables your Linux VM to use macOS rosetta. That allows you to run x86_64 applications in your Linux VM.

Besides checking this option you also need to run some command in Linux, you can find those commands in the UTM documentation you can also google it.

**Reply by myshkin5** [reply]
Yes and the docs you refer to are in the link @osy posted. Just not very clear from the dialog nor [linked install instructions](https://docs.getutm.app/advanced/rosetta/) on the dialog.
